include/triton/core/tritonserver.h

```diff
@@ -1761,6 +1761,14 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
     TRITONSERVER_ServerOptions* options, unsigned int thread_count);
 
+/// Provide a log output file.
+///
+/// \param options The server options object.
+/// \param file a string defining the file where the log outputs will be saved.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_ServerOptionsSetLogOutFile(
```

Comment:
`TRITONSERVER_ServerOptionsSetLogOutFile` -> `TRITONSERVER_ServerOptionsSetLogFile`

Reply:
I don't think this change belongs to this PR, should remove.